pkgs/top-level/all-packages.nix

```diff
@@ -14784,7 +14784,7 @@ in
 
     ati_drivers_x11 = callPackage ../os-specific/linux/ati-drivers { };
 
-    blcr = callPackage ../os-specific/linux/blcr { };
+    blcr = if builtins.compareVersions "3.18" kernel.version == 1 then callPackage ../os-specific/linux/blcr { } else null;
```

Comment:
Why not `lib.{versionOlder,versionAtLeast}`?

Reply:
fixed